src/FakeItEasy/ExceptionMessages.resx

```diff
@@ -208,4 +208,7 @@ The following constructors failed:
   <data name="FakeManagerWasInitializedWithDifferentProxyMessage" xml:space="preserve">
     <value>The fake manager was initialized for a different proxy.</value>
   </data>
+  <data name="NotRecognisedAsFake" xml:space="preserve">
+    <value>{0} is not recognized as a fake object.</value>

```

Comment:
Disregard my comment. Superseded by last bits of https://github.com/FakeItEasy/FakeItEasy/pull/1007/files#r105149664